tests/test_dcos_e2e/backends/aws/test_aws.py

```diff
@@ -274,12 +257,13 @@ def test_install_dcos_from_node(
             agents=0,
             public_agents=0,
         ) as cluster:
-            (master, ) = cluster.masters
-            master.install_dcos_from_url(
+            cluster.install_dcos_from_url(
                 build_artifact=oss_artifact_url,
                 dcos_config=cluster.base_config,
-                role=Role.MASTER,
                 log_output_live=True,
                 ip_detect_path=cluster_backend.ip_detect_path,
+                # This triggers a ``NotImplementedError`` which then which then
+                # leads to a ``Node.install_dcos_from_url`` installation.
+                files_to_copy_to_genconf_dir=[(Path('foo'), Path('/genconf'))],
```

Comment:
This is an implementation detail.
I think that there should be two tests:

* `test_install_dcos_from_node`
* `test_install_with_ files_to_copy_to_genconf_dir `